src/dlgProfilePreferences.cpp

```diff
@@ -1503,7 +1503,7 @@ void dlgProfilePreferences::setDisplayFont()
     }
     QFont font = fontComboBox->currentFont();
     font.setPointSize(mFontSize);
-    if (pHost->getDisplayFont() != font && pHost->setDisplayFont(font)) {
+    if (auto [setNewFont, errorMessage] = pHost->setDisplayFont(font); pHost->getDisplayFont() != font && setNewFont) {
```

Comment:
This makes the first condition obsolete because you set the font before you check if it is different. Also, why don't you use the error message here? My point from earlier still stands: Mudlet will silently do nothing.

Reply:
Just haven't gotten around to showing the message yet, sorry!